client/blocks/product-purchase-features/product-purchase-features-list/advertising-removed.jsx

```diff
@@ -14,7 +14,7 @@ export default localize( ( { isBusinessPlan, translate } ) => {
 		<div className="product-purchase-features-list__item">
 			<PurchaseDetail
 				icon="speaker"
-				title={ translate( 'Advertising Removed' ) }
+				title={ translate( 'Advertising removed' ) }
```

Comment:
Warning: 21 translations will be lost with this change.
Alternate string suggestion: 
* `translate( 'Remove all WordPress.com advertising from your website' )`&mdash; translations: **19**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/remove-live-course-advertising)